metadata.yaml

```diff
@@ -0,0 +1,21 @@
+name: metallb
+summary: |
+  This charm deploys MetalLB in a Kubernetes model, which provides a software
+  defined load balancer.
+docs: https://discourse.charmhub.io/t/metallb/6320
```

Comment:
I went ahead and pointed this at the docs page that the current metallb-controller and metallb-speaker charms use for now. But that page will need to be updated.

Of course, if I update that page now, it will no longer be correct for current stable charms.

@evilnick I really need a good way to stage doc changes for this, to go live at a later date. Any recommendations?